centreon/tests/e2e/features/Agent-configuration/05-agent-configuration-check-permissions/index.ts

```diff
@@ -0,0 +1,313 @@
+  cy.contains('Telegraf').click();
+  cy.getByLabel({ label: 'Name', tag: 'input' }).type('telegraf-001');
+  cy.getByLabel({ label: 'Pollers', tag: 'input' }).click();
+  cy.contains('Central').click();
+  cy.contains('Poller-1').click();
+  cy.getByLabel({ label: 'Public certificate file name', tag: 'input' }).type('my-otel-certificate-name-001');
+  cy.getByLabel({ label: 'CA file name', tag: 'input' }).type('ca-file-name-001');
+  cy.getByLabel({ label: 'Private key file name', tag: 'input' }).eq(0).type('my-otel-private-key-name-001');
+  cy.getByLabel({ label: 'Port', tag: 'input' }).should('have.value', '1443');
+  cy.getByLabel({ label: 'Certificate file name', tag: 'input' }).type('my-certificate-name-001');
+  cy.getByLabel({ label: 'Private key file name', tag: 'input' }).eq(1).type('my-conf-private-key-name-001');
+  cy.getByTestId({ testId: 'SaveIcon' }).click();
+  cy.wait('@addAgents');
+});
+
+Given('the user has a filter on one of the pollers', () => {
+  cy.setUserTokenApiV1().executeActionViaClapi({
+    bodyContent: {
+    action: 'addfilter_instance',
+    object: 'ACLRESOURCE',
+    values: `All Resources;Poller-1`
+    }
+  });
+  cy.setUserTokenApiV1().executeActionViaClapi({
+    bodyContent: {
+    action: 'reload',
+    object: 'ACL',
+    values: ''
+    }
+  });
+});
+
+When('the user accesses the Agents Configuration page', () => {
+  cy.reload();
+  cy.wait('@getAgentsPage');
+});
+
+Then('the user can not view the agent configuration linked to the 2 pollers', () => {
+  cy.contains('Welcome to the poller/agent configuration page').should('be.visible');
+  cy.contains('telegraf-001-updated').should('not.exist');
+});
+
+When('the admin user updates the filtered pollers of the non-admin user', () => {
+  cy.setUserTokenApiV1().executeActionViaClapi({
+    bodyContent: {
+    action: 'addfilter_instance',
+    object: 'ACLRESOURCE',
+    values: `All Resources;Central`
+    }
+  });
+  cy.setUserTokenApiV1().executeActionViaClapi({
+    bodyContent: {
+    action: 'reload',
+    object: 'ACL',
+    values: ''
+    }
+  });
+});
+
+Then('the user can view the agent configuration linked to the pollers', () => {
+  cy.reload();
+  cy.get('*[role="rowgroup"]')
+    .should('contain', 'telegraf-001');
+  cy.get('*[role="rowgroup"]')
+    .should('contain', '2 pollers');
+});
+
+Then('a pop up is displayed with all of the agent configuration information with the 2 pollers', () => {
+  cy.contains('Update poller/agent configuration').should('be.visible');
+  cy.getByLabel({ label: 'Agent type', tag: 'input' }).should('have.value', 'Telegraf');
+  cy.getByLabel({ label: 'Name', tag: 'input' }).should('have.value', 'telegraf-001');
+  cy.get('[class^="MuiButtonBase-root MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-deletable MuiChip-deletableColorDefault MuiChip-filledDefault MuiAutocomplete-tag MuiAutocomplete-tagSizeSmall"]').eq(0).should('have.text', 'Central');
```

Comment:
You need to find something much more precise; it’s best practice not to leave selectors like this